tests/config/test_themes.py

```diff
@@ -213,3 +215,25 @@ def test_add_pygments_style(
     # Check for overrides(k,sd) and inheriting styles (kr)
     for style in expected_styles:
         assert style in urwid_theme
+
+
+# Validate 16-color-codes
+@pytest.mark.parametrize(
+    "color_depth, theme_name",
+    [
+        (16, "zt_dark"),
+        (16, "gruvbox_dark"),
+        (16, "gruvbox_light"),
+        (16, "zt_light"),
+        (16, "zt_blue"),
```

Comment:
It is useful to use an existing theme as a basis for a test, but I'm not sure we benefit from testing against all the existing themes.

Instead, I'd suggest exploring different `Color` sets, eg. ensuring that a range of names work, as well as what output one gets if multiple colors in `Color` are invalid.